### ISSUE TITLE
Error Code 4: Miscellaneous of TensorRT 8.2.5.1 when running parse_from_file on GPU rtx3090

### ISSUE THREAD

**Issue by VariableXX** [NVIDIA/TensorRT#3126]
## Environment

**TensorRT Version**:

**NVIDIA GPU**:

**NVIDIA Driver Version**:

**CUDA Version**:

**CUDNN Version**:

Operating System:

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

I try to convert mask-rcnn implemented by torchvision, but error occured:
[07/12/2023-17:22:24] [TRT] [E] [graphShapeAnalyzer.cpp::nvinfer1::builder::`anonymous-namespace'::ShapeNodeRemover::analyzeShapes::1285] Error Code 4: Miscellaneous (IShuffleLayer Reshape_982: reshape changes volume. Reshaping [25063500] to [1,500].)
[07/12/2023-17:22:24] [TRT] [E] ModelImporter.cpp:773: While parsing node number 352 [Reshape -> "onnx::Sigmoid_2436"]:
[07/12/2023-17:22:24] [TRT] [E] ModelImporter.cpp:774: --- Begin node ---
[07/12/2023-17:22:24] [TRT] [E] ModelImporter.cpp:775: input: "onnx::Reshape_2434"
input: "1002"
output: "onnx::Sigmoid_2436"
name: "Reshape_982"
op_type: "Reshape"
attribute {
  name: "allowzero"
  i: 0
  type: INT
}
doc_string: "D:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torchvision\\models\\detection\\rpn.py(258): filter_proposals\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torchvision\\models\\detection\\rpn.py(366): forward\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\nn\\modules\\module.py(1098): _slow_forward\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\nn\\modules\\module.py(1110): _call_impl\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torchvision\\models\\detection\\generalized_rcnn.py(154): forward\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\nn\\modules\\module.py(1098): _slow_forward\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\nn\\modules\\module.py(1110): _call_impl\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\jit\\_trace.py(118): wrapper\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\jit\\_trace.py(127): forward\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\nn\\modules\\module.py(1110): _call_impl\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\jit\\_trace.py(1166): _get_trace_graph\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\onnx\\utils.py(391): _trace_and_get_graph_from_model\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\onnx\\utils.py(440): _create_jit_graph\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\onnx\\utils.py(499): _model_to_graph\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\onnx\\utils.py(719): _export\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\onnx\\utils.py(118): export\nD:\\ProgramData\\Miniconda3\\envs\\mmdet\\lib\\site-packages\\torch\\onnx\\__init__.py(305): export\nD:\\python_code\\TorchVision_Maskrcnn\\TorchVision_Maskrcnn\\my_model\\deploy\\pth2onnx.py(1he 7): <module>\n"

[07/12/2023-17:22:24] [TRT] [E] ModelImporter.cpp:776: --- End node ---
[07/12/2023-17:22:24] [TRT] [E] ModelImporter.cpp:779: ERROR: ModelImporter.cpp:179 In function parseGraph:
[6] Invalid Node - Reshape_982
[graphShapeAnalyzer.cpp::nvinfer1::builder::`anonymous-namespace'::ShapeNodeRemover::analyzeShapes::1285] Error Code 4: Miscellaneous (IShuffleLayer Reshape_982: reshape changes volume. Reshaping [25063500] to [1,500].)

The Reshape_982 in onnx is:
![fbfed7969013db2cacebfbf083728e3](https://github.com/NVIDIA/TensorRT/assets/42144172/2170a1d6-0cb6-4ea8-8ad3-cb502686604c)

that in code is:
![6485ecf9f50bd2ae08c4319169e7aa8](https://github.com/NVIDIA/TensorRT/assets/42144172/98ea6342-4ee7-4f25-8628-324c84a6886c)

It means that opeation of take tensor slice have three operators,  Reshape_982 is one of that. However, in onnx, the shape of Reshape_982 is clear, and onnx file is able to infer correctly.

convert code is：
```
import tensorrt as trt


def build_engine(onnx_file_path, engine_file_path, half=False):
    """Takes an ONNX file and creates a TensorRT engine to run inference with"""
    logger = trt.Logger(trt.Logger.INFO)
    builder = trt.Builder(logger)
    config = builder.create_builder_config()
    config.max_workspace_size = 4 * 1 << 30           
    flag = (1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH))
    network = builder.create_network(flag)
    parser = trt.OnnxParser(network, logger)
    if not parser.parse_from_file(str(onnx_file_path)):
        raise RuntimeError(f'failed to load ONNX file: {onnx_file_path}')
    half &= builder.platform_has_fast_fp16
    if half:
        config.set_flag(trt.BuilderFlag.FP16)
    with builder.build_engine(network, config) as engine, open(engine_file_path, 'wb') as t:
        t.write(engine.serialize())
    return engine_file_path
```

**Comment by zerollzeng** [comment]
Could you please provide the onnx here? Thanks!

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
